src/stream/src/executor/over_window/mod.rs

```diff
@@ -363,7 +362,9 @@ impl<S: StateStore> OverWindowExecutor<S> {
                 let (ret_values, evict_hints): (Vec<_>, Vec<_>) = partition
                     .states
                     .iter_mut()
-                    .map(|state| state.slide())
+                    .map(|state| state.output())
+                    .try_collect::<Vec<_>>()?
```

Comment:
What is the point of returning an error early here when .map(|o| (o.return_value, o.evict_hint)) has no side effects?

Reply:
Because batch aggregator will possibly throw an error

Reply:
Agree, but the collect is unnecessary if we have https://github.com/rust-itertools/itertools/issues/402

We can resolve it later :)